detection_rules/etc/_config.yaml

```diff
@@ -1,8 +1,8 @@
 # detection-rules config file
 
 rule_dirs:
-  - rules
-  - rules_building_block
+  - ../../rules
```

Comment:
what was happening before this change

Reply:
It basically would set the prebuilt `rules` path in an "etc" dir